src/shared/ui/Items/Text/index.tsx

```diff
@@ -4,16 +4,20 @@ type Props = {
   value: string | undefined;
   onValueChange: (value: string) => void;
   disabled: boolean;
+  isLarge?: boolean;
```

Comment:
minor nit but I think renaming to `isMultiline` would work better instead of `large`, as that could be confused by `size` or a different style